.gitignore

```diff
@@ -32,3 +32,7 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+#ignore rubocop files
+.rubocop.yml
+.rubocop_todo.yml
```

Comment:
what is this file? If you need to ignore files locally you can add them to your personal `~/.gitignore` (home directory)